source/_lovelace/calendar.markdown

```diff
@@ -35,6 +37,10 @@ title:
   required: false
   description: The title of the card.
   type: string
+initial_view:
+  required: false
+  description: The view that will show first when the card is loaded onto the page. Options are dayGridMonth, dayGridDay, and listWeek.
```

Comment:
```suggestion
  description: The view that will show first when the card is loaded onto the page. Options are `dayGridMonth`, `dayGridDay`, and `listWeek`.
```

Reply:
```suggestion
  description: "The view that will show first when the card is loaded onto the page. Options are `dayGridMonth`, `dayGridDay`, and `listWeek`."
```

Reply:
Needed to add quotes around the text to do this